core/src/main/java/com/linecorp/armeria/common/HttpHeaderNames.java

```diff
@@ -388,6 +388,18 @@
      */
     public static final AsciiString CONTENT_SECURITY_POLICY_REPORT_ONLY =
             create("Content-Security-Policy-Report-Only");
+    /**
+     * The HTTP/MIME {@code "Content-ID"} header field name.
+     */
+    public static final AsciiString CONTENT_ID = create("Content-ID");
```

Comment:
Could you sort the fields?

Reply:
I assume to sort in between the `CONTENT_...` names?
Done